tools/cmdautotest/README_CMDAutoTest.md

```diff
@@ -3,7 +3,7 @@
 The CMD autotest automatically performs the tasks you need to perform when performing multiple tests on the TASH command and determines the results.
 
 ### Prerequisites
-Install python2.7, pyserial(at pip)
+Install python3, pyserial(at pip3)
 
 ### How to USE
 $ python cmdautotest.py serial_port
```

Comment:
python or python3?

Reply:
Yes, it is exactly python3.x version, but abbreviated as python3.

Reply:
Support for Python version 2 is coming to an end. Therefore, the syntax is modified to version 3.